rotkehlchen/rotkehlchen.py

```diff
@@ -193,8 +193,8 @@ def maybe_kill_running_tx_query_tasks(
             for greenlet in self.api_task_greenlets:
                 is_evm_tx_greenlet = (
                     len(greenlet.args) >= 1 and
-                    isinstance(greenlet.args[0], MethodType) and
-                    greenlet.args[0].__func__.__qualname__ == 'RestAPI._get_evm_transactions'
+                    isinstance(greenlet.args[0], FunctionType) and
+                    greenlet.args[0].__qualname__ == 'RestAPI.get_evm_transactions'
```

Comment:
do we have tests to verify that this change doesn't break the logic?

Reply:
Yes, there is a test.

https://github.com/rotki/rotki/blob/e3a0c09c83eae4cb8ed28d676b7d60cc4107b3bd/rotkehlchen/tests/api/blockchain/test_evm.py#L299